### PR TITLE
Updated Readme, Build Passing with Travis CI!

### PR DESCRIPTION
**Pull request by ojlamb**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,6 @@
 # If you find yourself ignoring temporary files generated by your text editor
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
-/config/initializers/pusher.rb
```
Comment (zlahham) [inline .gitignore]: @ojlamb don't we actually need this to be gitignored?

File: app/assets/javascripts/viewer.js [hunk 2]
```diff
@@ -2,26 +2,21 @@
...
 
-
   Pusher.log = function(message) {
     if (window.console && window.console.log) {
       window.console.log(message);
     }
   };
 
-  var pusher = new Pusher('9a2f3193a0af5c40e9c0', {
+  var pusher = new Pusher('da575b9edde9ebf3a600', {
```
Comment (zlahham) [inline app/assets/javascripts/viewer.js]: @ojlamb this is not an issue, but the key needs to be the production one

File: .gitignore (hunk 1 above)
Comment (ojlamb) [inline .gitignore]: Travis needs them to run the the as well. I'v set the secret to an ENV variable though travis.

**Comment by zlahham** [comment]
Close #40

**Comment by zlahham** [comment]
Close #40

**Pull request merged**